### PR TITLE
refactor: publish helm sidam vars & Spring 2.4.5

### PR DESCRIPTION
**Pull request by abhijitdiwan**
### JIRA link (if applicable) ###
NA

### Change description ###
refactor: publish helm sidam vars & Spring 2.4.5

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -55,3 +55,16 @@ launchdarkly:
     environment: ${ENV_NAME:local}
     user: rd
```
Comment (AdilOoze) [inline src/main/resources/application.yaml:57]: You'll need to add `  config:
    import: "optional:configtree:/mnt/secrets/rd/"` to `application.yaml` as done in [this PR](https://github.com/hmcts/rd-professional-api/pull/1038/files)

Comment (abhijitdiwan) [reply src/main/resources/application.yaml:59]: done

File: charts/rd-judicial-data-load/values.yaml [hunk 2]
```diff
@@ -14,6 +14,8 @@ job:
     JRD_MQ_TOPIC_NAME: rd-judicial-topic-{{ .Values.global.environment }}
     JRD_MQ_TRUST_ALL_CERTS: false
     JRD_DATA_PER_MESSAGE: 500
+    IDAM_URL: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net
+    OAUTH2_REDIRECT_URI: https://rd-professional-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/oauth2redirect
```
Comment (SujeethKumarAri) [inline charts/rd-judicial-data-load/values.yaml:18]: I think this should not be professional-api

File: src/functionalTest/resources/application-integration.yml [hunk 3]
```diff
@@ -148,3 +148,17 @@ launchdarkly:
...
+elastic:
+  search:
+    query: ${SEARCH_QUERY_FROM:(roles:judiciary) AND lastModified:>now-72h}
+    recordsPerPage: 500
+
+idam:
+  api:
+    url: ${IDAM_URL:https://idam-api.aat.platform.hmcts.net}
+    client-id: ${OAUTH2_CLIENT_ID:rd-professional-api}
+    redirect-uri: ${OAUTH2_REDIRECT_URI:https://rd-professional-api-aat.service.core-compute-aat.internal/oauth2redirect}
```
Comment (SujeethKumarAri) [inline src/functionalTest/resources/application-integration.yml:161]: I think this should not be professional-api

File: src/functionalTest/resources/application-integration.yml [hunk 4]
```diff
@@ -148,3 +148,17 @@ launchdarkly:
...
+
+elastic:
+  search:
+    query: ${SEARCH_QUERY_FROM:(roles:judiciary) AND lastModified:>now-72h}
+    recordsPerPage: 500
+
+idam:
+  api:
+    url: ${IDAM_URL:https://idam-api.aat.platform.hmcts.net}
+    client-id: ${OAUTH2_CLIENT_ID:rd-professional-api}
```
Comment (SujeethKumarAri) [inline src/functionalTest/resources/application-integration.yml:160]: I think this should not be professional-api

File: src/main/resources/application.yaml [hunk 5]
```diff
@@ -55,3 +57,16 @@ launchdarkly:
...
 
+elastic:
+  search:
+    query: ${SEARCH_QUERY_FROM:(roles:judiciary) AND lastModified:>now-72h}
+    recordsPerPage: 500
+
+idam:
+  api:
+    url: ${IDAM_URL:https://idam-api.aat.platform.hmcts.net}
+    client-id: ${OAUTH2_CLIENT_ID:rd-professional-api}
```
Comment (SujeethKumarAri) [inline src/main/resources/application.yaml:68]: I think this should not be professional-api

File: src/main/resources/application.yaml [hunk 6]
```diff
@@ -55,3 +57,16 @@ launchdarkly:
...
+    redirect-uri: ${OAUTH2_REDIRECT_URI:https://rd-professional-api-aat.service.core-compute-aat.internal/oauth2redirect}
```
Comment (SujeethKumarAri) [inline src/main/resources/application.yaml:69]: I think this should not be professional-api

**Review by SujeethKumarAri — CHANGES_REQUESTED** [review]

File: charts/rd-judicial-data-load/values.yaml (hunk 2 above)
Comment (abhijitdiwan) [reply charts/rd-judicial-data-load/values.yaml:18]: :)

File: src/functionalTest/resources/application-integration.yml (hunk 4 above)
Comment (abhijitdiwan) [reply src/functionalTest/resources/application-integration.yml:160]: for now let it be as JRD-system-user not yet activated. hence using rd-professionla-api now.and it is fine for now

**Review by SujeethKumarAri — DISMISSED** [review]

**Review by shreedhar-hmcts — DISMISSED** [review]

**Review by shreedhar-hmcts — APPROVED** [review]

**Review by SujeethKumarAri — APPROVED** [review]

**Pull request merged**